docs/kane-ai-api-testing.md

```diff
@@ -0,0 +1,112 @@
+  
+
+## 3. Validating API Response on KaneAI
+
+To ensure the API works as expected, use the validation feature. This step confirms that the API responds correctly and can be added to test steps.
+
+- **Step**: Click the 'validate' option to check the API response. A 200 response status indicates successful validation, automatically adding the API to your test steps. You can now proceed to submit multiple APIs simultaneously if needed.
+
+  <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image3.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+  
+
+## 4. Adding Non-Success APIs in Test Steps
+
+For APIs that do not return a 200 status, you can review the response body and manually add them to the test steps as required.
+
+- **Step**: If the API returns a 400 Bad Request or another error, it will not be added automatically. Review the response and add the API manually if needed.
+
+  <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image8.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+  
+
+## 5. Adding Multiple APIs in One Go
+
+KaneAI allows batch processing of multiple APIs to streamline testing. This feature is helpful for scenarios requiring the execution of several API calls in succession.
+
+- **Step**: Add multiple APIs by clicking the plus icon and selecting each API, or paste multiple curl commands to add them automatically to the test steps.
+
+  <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image9.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+  
+
+## 6. Handling Different HTTP Methods
+
+KaneAI supports various HTTP methods like POST, PUT, GET, and DELETE, allowing you to test diverse API interactions.
+
+- **Step**: Add APIs using different HTTP methods, such as a PUT or DELETE command. Validate each API as before, and if successful, they will be automatically included in the test steps.
+
+  
+    <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image6.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+  
+    <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image10.jpg').default} alt="kenai-jira integration" className="doc_img"/>
+
+## 7. Executing and Reviewing Test Steps
+
+Once all APIs are added, KaneAI enables simultaneous execution, with details available on methods used, response statuses, and execution times.
+
+- **Step**: Click to execute all added APIs in one go and review the response details for insights into API performance and data returned.
+
+    <img loading="lazy" src={require('../assets/images/kane-ai/knowledge-base/api-testing/image10.jpg').default} alt="kenai-jira integration" className="doc_img"/>
```

Comment:
This needs to be have image 13

Reply:
@Private1647 but you have only sent me 10 images,